### ISSUE TITLE
Add flatpak support

### ISSUE THREAD

**Issue by LonelyMusicDisc** [glassmancody/waza_lightfixes#4]
I kept getting this error for seemingly no reason:

> user@fedora:~/Downloads/wazalightfixes $ ./waza_lightfixes 
Disable Flickering 	= True
no config file found, aborting!

Only to realize it means `openmw.cfg` in the configs folder. In flatpak, it's inside `~/.var/app/org.openmw.OpenMW/config/openmw` which isn't in the code.

I would appreciate added flatpak support or better yet, a more verbose error (I kept thinking it meant waza_lightfixes.cfg was not found) and a way to set the config file manually, like with arguments (`--config`/`-c`)

**Comment by VeeTeg** [comment]
First off, Glassmancody, thank you!  Secondly, I would also support some type of Flatpak support.

**Comment by JKlessens** [comment]
Also seconded and thanks for going to the trouble of making this tool!

**Comment by tristan-orourke** [comment]
I agree with wanting a way to set the file location manually, but in the meantime I have a good workaround. Waza expects the config file to be in the folder `~/.config/openmw`. If you create a symlink there, pointing to the real location of openmw.cfg, waza_lightfixes will run just fine.
